content/updates/serenity-updates-for-the-pim.json

```
@@ -0,0 +1,158 @@
+{
+    "data": [
+        {
+            "startDate": "June, 4th 2020",
+            "notification_duration": 14, 
+            "tags": ["new","updates"],
+            "title": "Easily monitor errors on your connections",
+            "description": "For each of your connections, a new `Monitoring` page now lists the last integration errors that may have occurred.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/new-connection-monitoring-page.png",
+            "altImg": "Connection monitoring page",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#easily-monitor-errors-on-your-connections"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notification_duration": 14, 
+            "tags": ["new","updates"],
+            "title": "New metrics on the Connection dashboard",
+            "description": "The Connection dashboard now displays additional information to ease error monitoring and allow you to see at a glance how your source connections are performing.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/error-monitoring-connection-dashboard.png",
+            "altImg": "Error monitoring on the Connection dashboard",
+            "link": "https://help.akeneo.com/pim/serenity/updates/2020-05.html#new-metrics-on-the-connection-dashboard"
+        },
+        {
+            "startDate": "June, 4th 2020",
+            "notification_duration": 14, 
+            "tags": ["new","updates"],
+            "title": "New association type: the two-way association",
+            "description": "Easily handle products compatibility in your PIM thanks to our **brand new two-way association type**! This new association type is now available under `Settings`/`Association types`.",
+            "img": "/bundles/akeneocommunicationchannel/images/cards/Associations_2-way-association-type.png",
```

@NolwennP I don't have this image in the zip you sent me yesterday

---

It's better if we name it with dash "-" instead of underscore "_" between words
```suggestion
            "img": "/bundles/akeneocommunicationchannel/images/cards/Associations-2-way-association-type.png",
```

---

My bad, I'll send it to you. ;)